hubgrep/lib/hosting_service_interfaces/gitea.py

```diff
@@ -121,31 +121,36 @@ def __init__(self, search_result_item, host_service_id):
 class GiteaSearch(HostingServiceInterface):
     name = "Gitea"
 
-    def __init__(self, host_service_id, api_url, requests_session=None):
+    def __init__(self, host_service_id, api_url, requests_session=None, timeout=2):
```

Comment:
redundant default as you already have it in the baseclass

Reply:
needed, because we overload the init method of the base class here. somewhere we have to pass the arg.